### PR TITLE
Add unique container names for publishing flow

### PR DESCRIPTION
**Pull request by tosan88**
Without kafka, kafka-rest-proxy & zookeper - separate PR will follow

### CODE REVIEW

**Comment by kovacshuni** [comment]
+1

**Comment by tosan88** [comment]
Tested in xp. Note: PAM and SIPM will alert as CRITICAL (as expected)

**Comment by catalin-ilea** [comment]
+1

File: binary-ingester-sidekick@.service [hunk 1]
```diff
@@ -12,11 +12,12 @@ ExecStart=/bin/sh -c "\
...
   while [ -z $PORT ]; do \
     sleep 5; \
-    PORT=`echo $(/usr/bin/docker port $SERVICE-%i 8080 | cut -d':' -f2)`; \
+    CONTAINER_NAME=$(docker ps -q --filter=name=\"$SERVICE\"-%i_); \
+    PORT=`echo $(/usr/bin/docker port $CONTAINER_NAME 8080 | cut -d':' -f2)`; \
   done; \
   etcdctl set /ft/services/$SERVICE/servers/%i http://%H:$PORT;"
 
-ExecStop=/usr/bin/etcdctl rm /services/binary-ingester/%i
+ExecStop=-/bin/bash -c '/usr/bin/etcdctl rm /services/binary-ingester/%i'
```
Comment (BerniVarga) [inline binary-ingester-sidekick@.service]: Shouldn't this be ft/services?

File: binary-writer-sidekick@.service [hunk 2]
```diff
@@ -12,11 +12,12 @@ ExecStart=/bin/sh -c "\
...
     sleep 5; \
-    PORT=`echo $(/usr/bin/docker port $SERVICE-%i 8080 | cut -d':' -f2)`; \
+    CONTAINER_NAME=$(docker ps -q --filter=name=\"$SERVICE\"-%i_); \
+    PORT=`echo $(/usr/bin/docker port $CONTAINER_NAME 8080 | cut -d':' -f2)`; \
   done; \
   etcdctl set /ft/services/$SERVICE/servers/%i http://%H:$PORT;"
 
-ExecStop=/usr/bin/etcdctl rm /services/binary-writer/%i
+ExecStop=-/bin/bash -c '/usr/bin/etcdctl rm /services/binary-writer/%i'
 Restart=on-failure
```
Comment (BerniVarga) [inline binary-writer-sidekick@.service]: rm `/ft/services/binary-writer/servers/%i`

File: binary-ingester-sidekick@.service (hunk 1 above)
Comment (tosan88) [inline binary-ingester-sidekick@.service]: Done!

File: binary-writer-sidekick@.service (hunk 2 above)
Comment (tosan88) [inline binary-writer-sidekick@.service]: Done!

**Comment by BerniVarga** [comment]
+1

**Pull request merged**